index.html

```diff
@@ -0,0 +1,182 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="reset.css">
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <title>Ghost Host Homepage</title>
+    </head>
+    <body>
+        <header>
+            <div class="header">
+            <div class="menu">
+                <div id="headerLogo"><img src="images/Header/Logo/Logo@1X.png" alt="Logo"></div>
+                <div id="menuBar">
```

Comment:
Please use `nav` + `a` tag for navigation 

https://css-tricks.com/navigation-in-lists-to-be-or-not-to-be/